ballerina/resiliency_failover_client.bal

```diff
@@ -73,7 +73,7 @@ public client isolated class FailoverClient {
     # + message - An HTTP outbound request or any allowed payload
     # + headers - The entity headers
     # + mediaType - The MIME type header of the request entity
-    # + targetType - HTTP response or `anydata`, which is expected to be returned after data binding
+    # + targetType - HTTP response, `anydata` or SseEvent `stream`, which is expected to be returned after data binding
```

Comment:
Are we enabling SSE events on top of resilient clients?

Reply:
Yes.